src/neighbors.cc

```diff
@@ -1145,7 +1145,10 @@ int
 neighborUp(const CachePeer * p)
 {
     if (!p->tcp_up) {
-        peerProbeConnect(const_cast<CachePeer*>(p));
+        // TODO: add and pass the CachePeer context here instead of nullptr
```

Comment:
To clarify that the CodeContext addition should not happen "here":

```suggestion
        // TODO: When CachePeer gets its own CodeContex, pass that context instead of nullptr
```